src/components/ErrorBoundary.tsx

```diff
@@ -53,6 +56,9 @@ export const ErrorBoundary = ({
     Sentry.init(sentryInit || {
       dsn: sentryDsn,
       environment: window.location.hostname,
+      initialScope: {
+        tags: { userData: (window as WindowWithUserData)._OX_USER_DATA?.uuid }
```

Comment:
am i reading the docs wrong? i see the example as
```
initialScope: {
    tags: { "my-tag": "my value" },
    user: { id: 42, email: "john.doe@example.com" },
  },
```
with user outside of tags, and as `user` not `userData`, am i looking in the wrong place?

Reply:
Evidently I have been taking crazy pills, because there was a reason I wasn't putting it in the `user` field, but that doesn't seem to be a problem now.